cmake/BTApplication.cmake

```diff
@@ -14,6 +14,12 @@ FIND_PACKAGE(Qt6 6.7 REQUIRED COMPONENTS
     Quick
     QuickWidgets
 )
+IF(BUILD_TEXT_TO_SPEECH)
+    FIND_PACKAGE(Qt6 REQUIRED COMPONENTS
+        TextToSpeech
+    )
```

Comment:
It might be better to fit this `FIND_PACKAGE()` call one line, because it would be less than 80 characters and it is unlikely that new Qt components will be added here any time soon.

Reply:
Done